src/drivers/mod.rs

```diff
@@ -0,0 +1,39 @@
+pub trait Driver{
+  fn get_status(&self) -> isize;
+  fn set_status(&mut self, status: isize);
+  fn execute(&self, motion: &String);
+}
+
+pub struct DefaultDriver {
```

Comment:
Im using Default as a sort of when there is nothing else use this back up so that things can work without me implementing postgres, however I can move it to its own file.